package.json

```diff
@@ -36,11 +36,11 @@
   },
   "scripts": {
     "precommit": "./pre-commit.sh",
-    "start": "yarn run migration --env development && node --trace-warnings ./server/bin/www",
+    "start": "yarn run migration --env production && node --trace-warnings ./server/bin/www",
```

Comment:
seems like we need a `start-dev` and then `dev` can use `yarn run start-dev` or something? so we don't have to keep making this change + reverting it, and also so that 'how to run in dev' is self-documenting.

Reply:
Done. Updated the document also. 

I have reverted back changes done by Nikhil(#78 ) for local testing. 
![image](https://user-images.githubusercontent.com/69027795/202224767-0cd11b7f-900a-4896-a1ff-be17ada3cb06.png)